message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -20,6 +20,7 @@
    [cmr.message-queue.config :as config]
    [cmr.message-queue.services.queue :as queue])
   (:import
+   (cmr.message_queue.test ExitException)
```

Comment:
Oh that's super confusing. 😄 
The namespace looks exactly like something that would be under /test/cmr/message_queue/test/.

Carry on.